### ISSUE TITLE
upgrade to webpack 4 and @babel 7

### ISSUE THREAD

**Issue by opadilla** [joeldenning/single-spa-examples#67]
Would love to use this but reverting our existing codebase from latest versions of webpack and babel to older versions (webpack 3 and babel 6) is a tough sell

**Comment by frehner** [comment]
This is just an example; it's not recommend to build an actual app based on this repo. 

If you're looking for a good example to build off of, check out https://single-spa.js.org/docs/separating-applications.html#comparison and check out the simple-webpack-example or systemjs example. :)

**Comment by internettrans** [comment]
Agreed -- this demo repo is not the best one to base your code off of. https://gitlab.com/TheMcMurder/single-spa-portal-example/ is a better one.

With that said, I agree this repo could use some attention. Will leave this open for when we have the time to upgarde.

**Comment by dzienisz** [comment]
@joeldenning maybe you should link this "better" example in main project repo?
